lib/v2/fzdm/manhua/manhua.js

```diff
@@ -0,0 +1,51 @@
+const got = require('@/utils/got');
+const domain = 'manhua.fffdm.com';
+const host = `https://${domain}`;
+const { art } = require('@/utils/render');
+const path = require('path');
+
+const get_pic = (id, chapter, caches) => {
+    const url = `${host}/api/manhua/${id}/${chapter}`;
+    return caches.tryGet(url, async () => {
+        const response = await got.get(url);
+        const data = response.data;
+        const comicTitle = data.mhinfo.title;
+        const chapterTitle = data.title;
+        const pics = data.cont;
+        return { comicTitle, chapterTitle, pics };
+    });
+};
+
+module.exports = async (ctx) => {
+    const id = ctx.params.id;
+    const count = ctx.query.limit || 99999;
+    const cdnNum = ctx.params.cdn || 5;
+    let cdn = '';
+    if (!isNaN(parseFloat(cdnNum)) && 1 <= parseFloat(cdnNum) && parseFloat(cdnNum) <= 5) {
+        cdn = `https://p${cdnNum}.fzacg.com`;
+    } else {
+        cdn = `https://p5.fzacg.com`;
+    }
+
+    // 获取漫画清单
+    const response = await got.get(`${host}/api/manhua/${id}`);
+    const data = response.data;
+
+    const chapter_detail = await Promise.all(
+        data.mhlist.splice(0, count).map(async (item) => {
+            const pic_content = await get_pic(id, item.url, ctx.cache);
+            return {
+                title: pic_content.chapterTitle,
+                description: art(path.resolve(__dirname, '../templates/manhua.art'), { pic: pic_content.pics, cdn }),
+                link: `${host}/api/manhua/${id}/${item.url}`,
+                comicTitle: pic_content.comicTitle,
+            };
+        })
+    );
+    ctx.state.data = {
+        title: '风之动漫 - ' + chapter_detail[0].comicTitle,
```

Comment:
Maybe use `data.mhinfo.latest.title` instead as it looks like it's more reliable

Reply:
![image](https://user-images.githubusercontent.com/16175551/219045873-a598c949-5a8b-49bd-9f17-6612e16ed3d4.png)
the data is wrong sometime.